### ISSUE TITLE
Regarding pretrained model for generating pseudo-labels

### ISSUE THREAD

**Issue by jaiabhayk** [yaircarmon/semisup-adv#3]
Is it possible for you to provide the pre-trained model used to generate the pseudo-labels ?
